conf/robottelo.yaml.template

```diff
@@ -29,3 +29,4 @@ ROBOTTELO:
   SETTINGS:
     GET_FRESH: true
     IGNORE_VALIDATION_ERRORS: false
+  SHARED_RESOURCE_WAIT: 2
```

Comment:
looks like you're missing a newline here

Reply:
fixed, thank you.